starcheck/src/starcheck

```diff
@@ -3,12 +3,12 @@
 unset LD_LIBRARY_PATH
 unset SYBASE
 unset PYTHONPATH
-SKA_bin=`dirname $0`
-eval `${SKA_bin}/flt_envs -shell sh`
+eval `${SKA}/bin/flt_envs -shell sh`
```

Comment:
The easiest thing might be to have a new, even-lighter-weight SKA/bin/starcheck wrapper  to call our starcheck shell script (maybe with a new name).